cmd/antrea-agent/agent.go

```diff
@@ -238,9 +238,17 @@ func run(o *Options) error {
 	}
 
 	var egressController *egress.EgressController
+	var nodeIP net.IP
+	if nodeConfig.NodeIPv4Addr != nil {
+		nodeIP = nodeConfig.NodeIPv4Addr.IP
+	} else if nodeConfig.NodeIPv6Addr != nil {
+		nodeIP = nodeConfig.NodeIPv6Addr.IP
+	} else {
+		return fmt.Errorf("NodeIPAddr in Node config invalid: %v", nodeConfig)
+	}
```

Comment:
dumb question: the commit message talks about dual-stack support but it seems that in a dual-stack cluster, Egress will only support the IPv4 address family?

Reply:
In a dual cluster, the IPv4 NodeIP just used to join memberlist cluster. It also can work if join with an IPv6 NodeIP.
It supports to create ExternalIPPool with IPv6 IPRange and assign it to Egress Node, which has been implemented in #2196 .

Reply:
Thanks I see that now